### DISCUSSION TITLE
How about Solidjs?

### DISCUSSION THREAD

**Discussion by neurocmd** [uNmAnNeR/imaskjs#631] [General]
How about porting imask to Solidjs?

**Comment by uNmAnNeR** [comment]
nice idea! do you have some example to start from? may be some libraries with similar behaviour?

**Reply by neurocmd** [reply]
Unfortunately, at the moment there are very few ready-made components. They are listed [here](https://github.com/one-aalam/awesome-solid-js#ui-components). 
But I created an example with a imask as a POC in [codesandbox](https://codesandbox.io/s/still-bush-lx0p7m?file=/src/IMaskInput.jsx)

**Reply by uNmAnNeR** [reply]
it's a great start already! is it fine if i create draft version of the plugin and request your review?

**Reply by uNmAnNeR** [reply]
@zamanruhy what do you think about using `use` directive to apply mask? https://github.com/davedbase/solid-slider#example

**Reply by neurocmd** [reply]
> it's a great start already! is it fine if i create draft version of the plugin and request your review?

Yes, it is
